Modules/zoneinfomodule.c

```diff
@@ -0,0 +1,2695 @@
+#include "Python.h"
+#include "structmember.h"
+
+#include <ctype.h>
+#include <stddef.h>
+#include <stdint.h>
+
+#include "datetime.h"
+
+// Imports
+PyObject *io_open = NULL;
+PyObject *_tzpath_find_tzfile = NULL;
+PyObject *_common_mod = NULL;
```

Comment:
You may declare them "static".

Also, it may be interesting to try to move these globals into a module state. But that can be done later.